extensions/find-permissions/main.ts

```diff
@@ -89,11 +89,11 @@ for (const path of requiredPaths) {
 console.log("]");
 
 // Recursively check the path for required sandboxing exceptions.
-async function checkPath(allowed: [string], path: string) {
+async function checkPath(allowed: string[], path: string) {
```

Comment:
I've seen this a few times now. So to clarify: `[string]` is an array with exactly one item (which is a `string`). This is the [tuple type](https://www.typescriptlang.org/docs/handbook/2/objects.html#tuple-types) in TypeScript. Very different than `string[]`, which is an array of `string`s

Reply:
Good catch, I wasn't aware of that.